.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,24 +1,31 @@
-<!-- Describe your pull request here -->
+**Describe your pull request here**
 
 ----
 
 Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).
 
-* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title
-* New recipes not directly relevant to the biological sciences need to be submitted to [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda
-* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please add the `please review & merge` label.
+* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
+* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
+* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
 * Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.
 
 <details>
-  <summary>BiocondaBot commands</summary>
-      
+  <summary>Please make use of the following BiocondaBot commands whenever appropriate:</summary>
```

Comment:
Advertising ;).

Reply:
Just a single day has passed and the text is already getting longer again ... ;-)

I like the colon though because it invites people to click on the 'expand' triangle.

How about:
`<summary>Available BiocondaBot commands:</summary>`

Reply:
Or at least "Please use the following ..." ("make use of" is just a wordy version of "use")

Reply:
> Just a single day has passed and the text is already getting longer again
> "make use of" is just a wordy version of "use"

Gotta love my inconsistency in saying "make it more succinct" in one place and not the other \*faceplam\*.
I think "Please use the following BiocondaBot commands:" should be fine. In theory, I'd still like to communicate the `whenever appropriate` part to emphatically tell people to use those commands -- but the "imperatively suggesting" `Please use` should be enough and is more concise.